pupil-spa/src/app/check/check.component.ts

```diff
@@ -45,7 +46,8 @@ export class CheckComponent implements OnInit {
               private warmupQuestionService: WarmupQuestionService,
               private auditService: AuditService,
               private storageService: StorageService,
-              protected windowRefService: WindowRefService) {
+              protected windowRefService: WindowRefService,
+              protected appCountService: AppCountService) {
```

Comment:
I believe this can be private?

Reply:
@danwilliams it should have been yes but the service is not needed in the check component so it is now removed.